.buildkite/pipeline.yml

```diff
@@ -9,7 +9,7 @@ steps:
     plugins:
       - thedyrt/skip-checkout#v0.1.1:
           cd: /home/ubuntu/chromium/src
-    command: "git fetch --all && git reset --hard origin/$BUILDKITE_BRANCH && node replay_build_scripts/update-all-repos.mjs && docker build -t chromium-build-new - < Dockerfile.build && git status && node buildLinux.mjs && node replay_build_scripts/upload_build_artifacts.mjs"
+    command: "git fetch --all && git reset --hard origin/$BUILDKITE_BRANCH && node replay_build_scripts/update-all-repos.mjs && pushd ../backend && buck2 kill && buck2 build --console simple //chromium:chromium && popd && node replay_build_scripts/upload_build_artifacts.mjs"
```

Comment:
The `buck2 kill` is necessary for now because, without it, buck2 will always cache our chromium build.

What we should do in the long run is bust the cache by renaming the symlink every time so other targets can be cached.